[RU] [EN] Update Modding, Beatmap Ranking Procedure and add its translation (#4518)
ppy/osu-wiki: wiki/Modding/ru.md

# Моддинг

> **Моддинг** процесс проверки чьих-либо [карт](/wiki/Beatmaps) с целью ее улучшения. Пользователя, который пишет мод называют *моддером* и совокупность их постов на странице обсуждения карты *модом*.

Reviewer: ```suggestion
**Моддинг** — процесс проверки чьих-либо [карт](/wiki/Beatmaps) с целью их улучшения. Пользователя, который пишет мод, называют *моддером*, а его посты на странице обсуждения карты — *модом*.
```

> Чаще всего моды пишут для повышения качества карты, идущей в категорию [Ranked](/wiki/Beatmaps#ranked). Так как моддинг — довольно тщательный процесс, требующий долгого анализа карты, пользователи часто занимаются им, чтобы лучше понимать маппинг.

Reviewer: ```suggestion
Чаще всего моды пишут для повышения качества карты, идущей в категорию [Ranked](/wiki/Beatmaps#ranked). Так как моддинг — довольно тщательный процесс, требующий долгого анализа карты, пользователи часто модят, чтобы лучше понимать маппинг.
```

> Моды чаще всего пишут для того, чтобы улучшить качество карт, нацеленных на получение [статуса рейтинговой](/wiki/Beatmaps#рейтинговые). Так как моддинг довольно детальный процесс и требует длительного анализа, пользователи часто используют его, чтобы лучше понимать маппинг.

Reviewer: ```suggestion
Чаще всего моды пишут для повышения качества карты, идущей в категорию [Ranked](/wiki/Beatmaps#ranked). Так как моддинг — довольно тщательный процесс, требующий долгого анализа карты, пользователи часто занимаются им, чтобы лучше понимать маппинг.
```

## Поиск карт для моддинга

> Существуют тоны способов найти карты для моддинга, вот некоторые из самых популярных:

Reviewer: ```suggestion
Существует много способов найти карты для моддинга. Вот некоторые из них:
```

> - [Modding Queues](https://osu.ppy.sh/community/forums/60) (форум для очередей моддинга) — первое место, куда идут в поиске карт для моддинга, а также для получения модов. В этом форуме можно создать тему, которая будет работать как очередь: другие люди будут оставлять в ней ссылки на карты, для которых они хотят получить моды.

Reviewer: ```suggestion
- [Modding Queues](https://osu.ppy.sh/community/forums/60) (очереди моддинга) — первое место, куда идут за картами для моддинга, а также для получения модов. В этом форуме можно создать тему, которая будет работать как очередь: другие люди будут оставлять в ней ссылки на карты, на которые они хотят получить моды.
```

> - [Форум моддинг очередей](https://osu.ppy.sh/community/forums/60) является самой активной платформой для поиска карт для моддинга, а также получения модов на свои работы. Пользователь может создать свою личную *очередь* на форуме, в которой другие люди могут публиковать свои карты, на которые они хотят получить моды.

Reviewer: ```suggestion
- [Modding Queues](https://osu.ppy.sh/community/forums/60) (форум для очередей моддинга) — первое место, куда идут в поиске карт для моддинга, а также для получения модов. В этом форуме можно создать тему, которая будет работать как очередь: другие люди будут оставлять в ней ссылки на карты, для которых они хотят получить моды.
```

>   Владелец каждой очереди может иметь свои собственные правила по тому, какие карты и при каких условиях получат его мод. Очереди обычно открыты для двух видов модов:

Reviewer: ```suggestion
  У каждой очереди свои условия для моддинга. Обычно предлагаются два вида модов:
```

>   - **Normal mod (NM):** Мод при котором не требуется никакой дополнительной работы от маппера.
>   - **Mod for mod (M4M):** В таких случаях моддер просит маппера также написать ему мод, оказывая услугу за услугу.

Reviewer: ```suggestion
  - **Normal mod (NM):** обыкновенный мод, не требующий от маппера никаких действий в отношении владельца очереди.
  - **Mod for mod (M4M):** мод, для получения которого нужно написать ответный мод на чью-нибудь карту (обговаривается заранее).
```

> - Известные моддеры часто указывают на то, принимают ли они просьбы на моддинг или нет, в их профилях.

Reviewer: ```suggestion
- Известные моддеры часто пишут, принимают они просьбы о моддинге или нет, в своих профилях.
```

> Просто так искать карты для моддинга в категории [Pending](/wiki/Beatmaps#Ожидающие) **не** рекомендуется: авторы большинства из них не планируют их ранкать, и ваш мод вряд ли будет применён.

Reviewer: ```suggestion
Просто так искать карты для моддинга в категории [Pending](/wiki/Beatmaps#pending) **не** рекомендуется: авторы большинства из них не планируют их ранкать, и ваш мод вряд ли будет применён.
```

> - Поиск карт с помощью библиотеки карт **не** рекомендуется, так как большинство загруженных карт не расчитаны на получение Ранкеда, из-за чего их создатели вряд-ли ответят на ваш мод.

Reviewer: вот тут в оригинале тоже логическая ошибка, которую нужно исправить (убрать пункт из списка): перечисляются способы поиска карт для моддинга, а не то, что не рекомендуется делать.
```suggestion

Просто так искать карты для моддинга в категории [Pending](/wiki/Beatmaps#pending] **не** рекомендуется: авторы большинства из них не планируют их ранкать, и ваш мод вряд ли будет применён.
```

## Как писать мод

> Загрузите самую последнюю версию карты и откройте её [страницу с обсуждением](/wiki/Beatmap_Discussion). Сыграйте [сложность](/wiki/Difficulties), для которой вы хотите написать мод, отмечая про себя найденные проблемы и недочёты. После этого откройте карту в редакторе и просмотрите её на наличие других проблем.

Reviewer: ```suggestion
Загрузите самую последнюю версию карты и откройте [страницу с её обсуждением](/wiki/Beatmap_Discussion). Сыграйте [сложность](/wiki/Difficulties), для которой вы хотите написать мод, отмечая про себя найденные проблемы и недочёты. После этого откройте карту в редакторе и проверьте её на наличие других проблем.
```

> Чтобы начать писать мод, загрузите самую последнюю версию карты и откройте ее [страницу с обсуждением](/wiki/Beatmap_Discussion). Сыграйте [сложность](/wiki/Difficulties), для которой вы хотите написать мод и запоминайте проблемы, замеченные при игре. Когда вы ее сыграли, откройте карту в редакторе и проанализируйте ее для поиска других проблем.

Reviewer: ```suggestion
Загрузите самую последнюю версию карты и откройте её [страницу с обсуждением](/wiki/Beatmap_Discussion). Сыграйте [сложность](/wiki/Difficulties), для которой вы хотите написать мод, отмечая про себя найденные проблемы и недочёты. После этого откройте карту в редакторе и просмотрите её на наличие других проблем.
```

> Когда вы обнаружили проблему, выделите связанные с ней объекты в редакторе, а затем скопируйте их таймкод, нажав `Ctrl` + `C`. Вставьте этот таймкод на странице обсуждения карты в одну из вкладок, нажав `Ctrl` + `V`, и напишите после него свои мысли и идеи. Если замечание связано с конкретными частями сложности, оставьте его на вкладке `Timeline`, если с обширными участками или всей сложностью в целом — на вкладке `General (This difficulty)`, а для всего, что применимо ко всем сложностям в карте, используйте вкладку `General (All difficulties)`. 

Reviewer: ```suggestion
Если вы обнаружили проблему, выделите в редакторе относящиеся к ней объекты, а затем скопируйте их таймкод, нажав `Ctrl` + `C`. Вставьте этот таймкод на странице обсуждения карты в одну из вкладок, нажав `Ctrl` + `V`, и напишите после него свои мысли и идеи. Если замечание связано с конкретными частями сложности, оставьте его на вкладке `Timeline`, если с обширными участками или всей сложностью в целом — на вкладке `General (This difficulty)`, а для вещей, применимых ко всем сложностям в карте, используйте вкладку `General (All difficulties)`. 
```

> Когда вы обнаружили проблему, выберите связанные с ней объекты в редакторе, а затем скопируйте их временной код как текст используя горячие клавиши (`Ctrl` + `C`). На странице обсуждения карты, вставьте (`Ctrl` + `V`) этот таймкод, а также ваше предложение. `Временная шкала` предназначена для проблем, связанных с конкретными частями карты, вкладка `Общее (эта сложность)` используется для описания предложений на бóльших секциях карты, а вкладка `Общее (все сложности)` является местом жительством для проблем и предложений, которые применимы ко всем сложностям в карте. 

Reviewer: ```suggestion
Когда вы обнаружили проблему, выделите связанные с ней объекты в редакторе, а затем скопируйте их таймкод, нажав `Ctrl` + `C`. Вставьте этот таймкод на странице обсуждения карты в одну из вкладок, нажав `Ctrl` + `V`, и напишите после него свои мысли и идеи. Если замечание связано с конкретными частями сложности, оставьте его на вкладке `Timeline`, если с обширными участками или всей сложностью в целом — на вкладке `General (This difficulty)`, а для всего, что применимо ко всем сложностям в карте, используйте вкладку `General (All difficulties)`. 
```

> Все посты в моде должны явно показывать что проблема, почему это проблема и как ее исправить. Прочитайте [Нормы поведения в моддинге и маппинге](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#making-a-mod-post) для того, чтобы узнать больше подробностей о том, как оформлять моды.

Reviewer: ```suggestion
Все моды должны явно указывать, что именно является недостатком, почему это — недостаток, и как его исправить. Более подробно о том, как оформлять мод-посты, написано в статье «[Нормы поведения в моддинге и маппинге](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#making-a-mod-post)».
```

> Мапперы регулярно дают [кудосу](/wiki/Glossary#kudosu!) мод постам, которые они посчитали полезными нажимая на *палец вверх* справа от поста. Количество кудосу сохраняется в профилях моддеров и их количество рассматривается, если моддер подаёт заявку на становление [номинатором карт](/wiki/People/The_Team/Beatmap_Nominators).

Reviewer: ```suggestion
Если маппер посчитал мод полезным, он может выдать его автору [кудосу](/wiki/Glossary#kudosu!), нажав на *большой палец вверх* справа от поста. Число полезных модов показано в профиле каждого игрока на вкладке `modding`; оно учитывается в случае подачи заявки в группу [номинаторов](/wiki/People/The_Team/Beatmap_Nominators).
```

## Виды постов в модах

> Цель каждой `Проблемы` или `Предложения` в обсуждении карты объяснить что-либо относительно карты в контексте ее дизайна. 

Reviewer: ```suggestion
Моды бывают двух видов: `Problem` (ошибка, которую нужно исправить) и `Suggestion` (то, что можно улучшить). В зависимости от сути замечания в адрес карты, можно использовать тот или иной вид. 
```

> Многие мод посты показывают ошибки в карте. Например, пропущенный звук хлопка, который должен был быть там учитывая логику расстановки хитсаундов в карте, скорее всего ошибка.

Reviewer: ```suggestion
Многие мод-посты указывают на ошибки в карте. Например, пропущенный звук хлопка, который должен быть на ноте, если следовать логике расстановки хитсаундов в карте, скорее всего, является ошибкой.
```

> Другой распространённый тип мод-поста — предложение. Участок карты может не иметь ошибок, но при этом её можно сделать ещё лучше. В этом случае моддер поможет раскрыть потециал карты, указав, как переделать такой участок.

Reviewer: ```suggestion
Другой распространённый тип мод-поста — предложение. Участок карты может не иметь ошибок, но при этом его можно улучшить. В этом случае моддер поможет раскрыть потециал карты, указав, что и как переделать.
```

> Предложение других путей, чтобы замапать какой-то кусок карты, также распространенный тип мод постов. В то время как часть карты может быть приемлемой, моддер может объяснить как ее улучшить изменив ее дизайн.

Reviewer: ```suggestion
Другой распространённый тип мод-поста — предложение. Участок карты может не иметь ошибок, но при этом её можно сделать ещё лучше. В этом случае моддер поможет раскрыть потециал карты, указав, как переделать такой участок.
```

> Карты, которые созданы для ранкеда, обязаны следовать [критериям ранкинга](/wiki/Ranking_Criteria). Моды часто упоминают проблемы, вытекающие из этих критериев.

Reviewer: ```suggestion
Кроме того, часть модов посвящена [критериям ранкинга](/wiki/Ranking_Criteria), которым должны следовать все карты, идущие на ранкинг. Данные моды также помечаются как указания на ошибки.
```
